Dockerfile

```diff
@@ -16,7 +16,6 @@ USER node
 RUN yo --no-insight @plone/volto my-volto-project --no-interactive
 
 COPY jsconfig docker-entrypoint.sh /
-COPY jest.config.js /opt/frontend/my-volto-project/
```

Comment:
We can have an empty default `jest-addon.config.js` here in case the add-on doesn't provide it.

Reply:
Done

Reply:
Done